tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -246,6 +246,7 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   pm.addNestedPass<FuncOp>(disc_ral::createDiscAlgebraSimplifierPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscSplitLargeOpsPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscDotRewriterPass());
+  pm.addNestedPass<FuncOp>(disc_ral::createDiscMhloRewriterPass());
```

Comment:
maybe use a more specific name? `mhlo` is too general.

Reply:
You have different opinions with @Yancey1989, hahaha. IMO it's ok here since we should have a place to write decomposition for mhlo ops. I think the pass could be.

Reply:
even using `decomposition` is better than `MHLO`? otherwise, the name of the pass is less self-explanatory.

Reply:
Renamed to `MhloDecompositionRewriter`